packages/automated-dispute/src/services/discordNotifier.ts

```diff
@@ -1,98 +1,117 @@
  */
 export class DiscordNotifier implements NotificationService {
-    private client: Client;
-    private config: DiscordNotifierConfig;
-    private logger: ILogger;
+    private client: WebhookClient;
 
-    private constructor(client: Client, config: DiscordNotifierConfig, logger: ILogger) {
-        this.client = client;
-        this.config = config;
-        this.logger = logger;
+    /**
+     * Creates an instance of DiscordNotifier.
+     *
+     * @param {string} url - The Discord webhook URL.
+     */
+    constructor(url: string) {
+        this.client = new WebhookClient({ url });
     }
 
     /**
-     * Creates an instance of the DiscordNotifier.
-     * @param {DiscordNotifierConfig} config - The configuration object for the DiscordNotifier.
-     * @param {ILogger} logger - The logger instance.
-     * @returns {Promise<DiscordNotifier>} A promise that resolves to a DiscordNotifier instance.
+     * Sends a notification message to Discord.
+     *
+     * @param {IMessage} message - The message to send.
+     * @returns {Promise<void>} A promise that resolves when the message is sent.
      */
-    public static async create(
-        config: DiscordNotifierConfig,
-        logger: ILogger,
-    ): Promise<DiscordNotifier> {
-        const intents = new IntentsBitField().add(
-            IntentsBitField.Flags.Guilds,
-            IntentsBitField.Flags.GuildMessages,
-        );
-        const client = new Client({ intents });
-
+    async send(message: IMessage): Promise<void> {
+        const webhookMessage = this.buildWebhookMessage(message);
         try {
-            await client.login(config.discordBotToken);
-
-            await new Promise<void>((resolve, reject) => {
-                client.once("ready", () => {
-                    logger.info("Discord bot is ready");
-
-                    resolve();
-                });
-
-                client.once("error", (error: Error) => {
-                    reject(error);
-                });
-            });
+            await this.client.send(webhookMessage);
+            console.error(this.client, this.client);
         } catch (error) {
-            logger.error(`Failed to initialize Discord notifier: ${error}`);
-
-            throw error;
+            throw new NotificationFailureException("An error occurred with the Discord client.");
         }
+    }
 
-        return new DiscordNotifier(client, config, logger);
+    /**
+     * Builds a Discord webhook message from the given IMessage.
+     *
+     * @param {IMessage} message - The message details.
+     * @returns {WebhookMessage} The built Discord webhook message.
+     */
+    private buildWebhookMessage(message: IMessage): WebhookMessage {
+        return {
+            username: message.username || "EBO Agent",
+            content: message.title,
+            avatarURL: message.avatarUrl || "https://cryptologos.cc/logos/the-graph-grt-logo.png",
```

Comment:
We want the URL to be a new config param (new value in the config yml called `notifier.discordDefaultAvatarUrl` which is a `string().url()` + `optional()` if Discord allows you to specify no avatar URL) to be set during the class constructor:
```suggestion
            avatarURL: message.avatarUrl || this.defaultAvatarUrl,
```